app/src/main/java/io/gnosis/safe/di/components/ApplicationComponent.kt

```diff
@@ -53,7 +54,9 @@ interface ApplicationComponent {
 
     fun termsChecker(): TermsChecker
 
-    fun screenshotAuthorizer(): PrivacySettingsHandler
+    fun privacySettingsHandler(): PrivacySettingsHandler
+
+    fun appearanceSettingsHandler(): AppearanceSettingsHandler
```

Comment:
PrivacySettingshandler and AppearanceSettingsHandler could be merged into SettingsHandler. What do you think?

Reply:
yes